contracts/tansu/src/contract_versioning.rs

```diff
@@ -50,15 +47,7 @@ impl VersioningTrait for Tansu {
             config: types::Config { url, ipfs },
             maintainers: maintainers.clone(),
         };
-        let str_len = name.len() as usize;
-        if str_len > 15 {
-            // could add more checks but handled in any case with later calls
-            panic_with_error!(&env, &errors::ContractErrors::InvalidDomainError);
-        }
-        let mut slice: [u8; 15] = [0; 15];
-        name.copy_into_slice(&mut slice[..str_len]);
-        let name_b = Bytes::from_slice(&env, &slice[0..str_len]);
-
+        let name_b = crate::get_name_bytes(&env, &name);
```

Comment:
No don't do any refactoring for a migration. Migrations are one-off scripts and must not drive the business logic. Same goes for tests. If we need to re-do something, it's perfectly fine. The other issue is that doing so breaks code lineage.

Reply:
Reverted, I was trying to avoid duplication.